salt/cdn-logs/init.sls

```diff
@@ -14,6 +14,16 @@
   file.managed:
     - source: salt://cdn-logs/config/fastly.logrotate.conf
 
+logrotate_time_hourly:
+  file.replace:
+    - name: /etc/systemd/system/timers.target.wants/logrotate.timer
+    - pattern: OnCalendar=daily
+    - repl: OnCalendar=hourly
+
 /etc/cron.hourly/logrotate:
   file.symlink:
     - target: /etc/cron.daily/logrotate
+  cmd.run:
```

Comment:
This command should run in response to the systemd configuration changing, not the cron configuration. I think it should be moved into the state above `logrotate_time_hourly`.